backend/src/modules/tournament-tables/core/domain/TournamentTable.ts

```diff
@@ -79,22 +85,22 @@ export function bookTournamentTable(
 
   const bookedTable = onTableAvailabilityWasChanged(state);
 
-  const tournamentTableWasBooked = new TournamentTableWasBooked({
-    occurredAt: currentTime,
-    tournamentId: bookedTable.tournamentId,
-    tableNumber: bookedTable.tableNumber.raw,
-  });
+  const tournamentTableWasBookedEvent = tournamentTableWasBooked(currentTime, bookedTable.tournamentId, bookedTable.tableNumber.raw);
 
   return {
     state: bookedTable,
-    events: [tournamentTableWasBooked],
+    events: [tournamentTableWasBookedEvent],
   };
 }
 
 function onTableAvailabilityWasChanged(state: TournamentTable): TournamentTable {
   return new TournamentTable({ ...state, isFree: !state.isFree });
 }
 
+function tournamentTableWasBooked(occurredAt: Date, tournamentId: string, tableNumber: number): TournamentTableWasBooked {
```

Comment:
Nie rozumiem tworzenia funkcji, która jedynie wywołuje konstruktor :) ? Jeszcze tracimy tutaj to co wprowadza nam props, czyli nazwane argumenty.
Chyba by wystarczyło 
```
const tournamentWasBooked = TournamentTableWasBooked({ occurredAt: currentTime, tournamentId: bookedTable.tournamentId, tableNumber: bookedTable.tableNumer.raw })
```

To tez wproawdzilo problemy, ze musialas zmienci nazwe zmiennej tournamentTableWasBooked na tournamentTableWasBookedEvent.